### ISSUE TITLE
Download rejected by Kaspersky

### ISSUE THREAD

**Issue by RealTOC** [LesFerch/WinSetView#18]
I tried downloading under Firefox via two routes.

When using the download button on the Code page, Kaspersky refused because of HEUR:Trojan.Script.Generic, on the basis of Heuristic Analysis, mentioning Expert Analysis.

I then clicked on the Latest button on the right and successfully downloaded the zip file.  However, when I extracted WinSetView.hta, Kaspersky immediately deleted it, giving the same reason.

**Comment by LesFerch** [comment]
It's a false positive. I have a case open with Kaspersky. In my testing, this only happens with Kaspersky Internet Security (not Total Security or Security Cloud -Free). I had the same issue with Windows Defender and it got fixed the same day I submitted the report, so, hopefully, Kaspersky will also fix this quickly.

Case Details:
ID INC000013011632
Request topic: False positive
File: WinSetView.hta
From: https://github.com/LesFerch/WinSetView
OS: Windows 10
Product: Kaspersky Internet Security
Note 1: WinSetView.hta scans clean at https://opentip.kaspersky.com
Note 2: WinSetView.hta was NOT blocked by Kaspersky Total Security or Security Cloud – Free
Note 3: The false positive only appears to occur with Kaspersky Internet Security
Note 4: Windows Defender definitions were updated to fix same false positive.
See: https://www.microsoft.com/en-us/wdsi/submission/2d75e0c4-e09b-451f-a875-082dcf6bca4c
